### PR TITLE
replace eslint with biome

### PR DESCRIPTION
**Pull request by cre8**
eslint is not designed to support monorepos, biome seems to work pretty well for this.

The job is not run via lerna but by bionme on the root level. This allows to follow the links printent in the console. It okay to lint every project independent if it has updated files since all projects should be compliant to linting rules.

### CODE REVIEW

**Comment by lukasjhan** [comment]
Some of code line failed with the current rules. Do you want to fix that later?

File: biome.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.5.3/schema.json",
+  "organizeImports": {
+    "enabled": true
+  },
+  "files": {
+    "ignore": ["**/dist/**"]
```
[the remark this answers is not in the document]
Comment (lukasjhan) [reply biome.json:7]: How about ignore tests? 
**/*.spec.ts

**Comment by lukasjhan** [comment]
linter works well, 30+ founds :D

File: biome.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (lukasjhan) [reply biome.json:7]: @cre8 You can check the example here: https://github.com/lukasjhan/sd-jwt-js/commit/cebb180ee33b68cedd52ec661cb4d9cfc90cb20e

**Comment by lukasjhan** [comment]
> @lukasjhan the intention of my first commits was to get feedback for the defined rules ;) E.g. some people don't like it if you use the ! or any type in the code, but forcing these rules can lead to bigger changes in the code

! and any. hmm... In my opinion, the rules in this PR is okay. Kinds of reasonable rules.

File: biome.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (cre8) [reply biome.json]: @lukasjhan I patched all files that got errors when running `pnpm run lint`, now the CI passes. Open question is the format one. I will check if we should also include the format validation to make sure people are not using ' and " in the code.

[the remark this answers is not in the document]
Comment (lukasjhan) [reply biome.json]: @cre8 they can only use ` in their code?

[the remark this answers is not in the document]
Comment (lukasjhan) [reply biome.json]: QuoteStyle forcing could be good idea. It makes PR less messy

[the remark this answers is not in the document]
Comment (lukasjhan) [reply biome.json]: I checked "quoteStyle": "single", its already applied. :)

**Review by berendsliedrecht — COMMENTED** [review]
Thanks for the really quick pick up! Really liking the look of the setup and how easy it seems :).

**Review by berendsliedrecht — APPROVED** [review]

File: biome.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      "quoteStyle": "single",
+      "lineEnding": "lf"
+    }
+  },
+  "linter": {
+    "enabled": true,
+    "rules": {
+      "recommended": true,
+      "suspicious": {
+        "noExplicitAny": "off"
```
[the remark this answers is not in the document]
Comment (cre8) [reply biome.json:25]: @berendsliedrecht I opened an issue for this #79 , feel free to work on this :)

**Pull request merged**